vignettes/Get_Started.Rmd

```diff
@@ -0,0 +1,212 @@
+---
+title: "Get Started"
```

Comment:
If you change the name of this file to `multilevelmod.Rmd` it will show up as "Get started" in the navbar.